helm/azuredisk-csi-driver-app/templates/azuredisk-controller/storage-class.yaml

```
@@ -0,0 +1,44 @@
+{{- if and .Values.provider (eq .Values.provider "capz") -}}
```

couldn't we also do a

```suggestion
{{- if (eq .Values.provider "capz") -}}
```

---

I wanted to make sure the `provider` key was set , i am not sure how helm will react if the `provider` key is not set ?